### PR TITLE
Feature/shared sdk initializer

### PR DESCRIPTION
**Pull request by jcaliksygic**

### CODE REVIEW

File: SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift [hunk 1]
```diff
@@ -71,7 +71,13 @@ public class SYMKBrowseMapViewController: UIViewController {
     
     override public func viewDidLoad() {
         super.viewDidLoad()
-        initSygicMapsSDK()
+        SYMKSdkManager.shared.initailizeIfNeeded { [weak self] success in
```
Comment (Kraci) [inline SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift]: initialize

File: SygicMapsKit/Classes/SYMKSdkManager.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import SygicMaps
+
+public typealias SdkInitializationCompletion = (_ success: Bool)->()
+
+public class SYMKSdkManager {
```
Comment (Kraci) [inline SygicMapsKit/Classes/SYMKSdkManager.swift:5]: Pridal by som `private init()` kedze je to singleton, alebo ked init nieje uvedeny sprava sa ako private?

Comment (jcaliksygic) [reply SygicMapsKit/Classes/SYMKSdkManager.swift:5]: private init, public shared

**Review by Kraci — APPROVED** [review]
👍 kuul

**Pull request merged**